Based on patches provided by John Dey from Fred Hutch:
https://github.com/FredHutch/easybuild-life-sciences/blob/main/fh_easyconfigs/c/CellProfiler/CellProfiler-4.2.4-foss-2021b.eb
Updated to reflect newer versions of some python packages
Uploaded: J. Sassmannshausen (Imperial College London/UK)
diff --git a/cellprofiler-core-4.2.6.orig/setup.py b/cellprofiler-core-4.2.6/setup.py
index 42b88fb..ccba16f 100644
--- a/cellprofiler-core-4.2.6.orig/setup.py
+++ b/cellprofiler-core-4.2.6/setup.py
@@ -22,17 +22,17 @@ setuptools.setup(
     },
     install_requires=[
         "boto3>=1.12.28",
-        "centrosome==1.2.2",
-        "docutils==0.15.2",
-        "h5py~=3.6.0",
+        "centrosome==1.2.1",
+        "docutils>=0.15.2",
+        "h5py>=3.6.0",
         "matplotlib>=3.1.3",
         "numpy>=1.18.2",
         "prokaryote==2.4.4",
         "psutil>=5.7.0",
         "python-bioformats==4.0.7",
         "python-javabridge==4.0.3",
-        "pyzmq~=22.3",
-        "scikit-image==0.18.3",
+        "pyzmq>=22.3",
+        "scikit-image>=0.18.3",
         "scipy>=1.4.1",
     ],
     license="BSD",
